modules/network/src/testFixtures/java/org/apache/ignite/internal/network/utils/ClusterServiceTestUtils.java

```diff
@@ -178,24 +177,12 @@ public CompletableFuture<Void> start() {
                                 )
                 ).join();
 
-                bootstrapFactory.start();
-
-                clusterSvc.start();
-
-                return nullCompletedFuture();
+                return allOf(bootstrapFactory.startAsync(), clusterSvc.startAsync());
             }
 
             @Override
-            public void stop() {
-                try {
-                    IgniteUtils.closeAll(
-                            clusterSvc::stop,
-                            bootstrapFactory::stop,
-                            nodeConfigurationMgr::stop
-                    );
-                } catch (Exception e) {
-                    throw new RuntimeException(e);
-                }
+            public CompletableFuture<Void> stopAsync() {
+                return allOf(clusterSvc.stopAsync(), bootstrapFactory.stopAsync(), nodeConfigurationMgr.stopAsync());
```

Comment:
It's not an equivalent of the original code. closeAll tolerates exceptions while closing each resource and thus best-effort.